.github/workflows/image.yml

```diff
@@ -27,6 +27,5 @@ jobs:
         pull: true
         push: true
         tags: |
-          quay.io/gatekeeper/operator:v0.0.1
           quay.io/gatekeeper/operator:latest
```

Comment:
We'll leave `latest` for now since there's no official release. Once we have our first release, we'll want to have `latest` point to the latest released version.